dom/schema/v1/jobimp.sql

```diff
@@ -0,0 +1,17 @@
+create table if not exists jobimp
```

Comment:
"jobimp" suffix in table name makes not much sense, why not just "job"

Reply:
Agree. But the incoming data format is not stabilized yet. Temporary name for now.